src/compiler/mir/ops/traverser.rs

```diff
@@ -156,7 +156,26 @@ impl<'a, L, V, T: FunctionBuilder<L, V>> FunctionTraverser<'a, L, V, T> {
                 let cond = self.operand(cond);
                 self.xfmr.term_cond_goto(cond, *tru, *fls)
             }
-            TerminatorKind::CallFn { .. } => todo!(),
+            TerminatorKind::CallFn {
+                func,
+                args,
+                reentry,
+            } => {
+                // Get the target function
+                let target = match func {
+                    Operand::Constant(_) => todo!(),
+                    Operand::LValue(l) => self.lvalue(l),
+                };
```

Comment:
Should we technically consider a call to a function, by name, as a constant?

Or should this panic?